src/core/ext/filters/client_channel/resolver_result_parsing.cc

```diff
@@ -111,8 +112,8 @@ ClientChannelServiceConfigParser::ParseGlobalParams(const ChannelArgs& /*args*/,
         lb_policy_name[i] = tolower(lb_policy_name[i]);
       }
       bool requires_config = false;
-      if (!LoadBalancingPolicyRegistry::LoadBalancingPolicyExists(
-              lb_policy_name.c_str(), &requires_config)) {
+      if (!lb_policy_registry.LoadBalancingPolicyExists(lb_policy_name.c_str(),
```

Comment:
Not new in this PR, but it looks like the `.c_str()` is no longer needed.

Reply:
ack... I'm going to leave it for now